egs2/slurp/slu1/local/data.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
```

Comment:
Will we replace this version with the old version with ASR? If no, better to use a symlink, if yes, please keep this one.

Reply:
Yes, this is different from one in asr because we are also creating transcript. Hence i kept this.